In the case of HTML Paragraph, erroneous detection occurs.

I applied the no-start-duplicated-conjunction rule to the HTML file.
The following Lint error occurred, but I think that this error is a false detection.

```console
  916:1011  error    Don't repeat "</p>" in 2 phrases    no-start-duplicated-conjunction
```

In the case of HTML, the data of the node of Paragraph is as follows.

```console
{ type: 'Paragraph',
  tagName: 'p',
  properties: {},
  children: 
   [ { type: 'Strong',
       tagName: 'strong',
       properties: {},
       children: [Array],
       loc: [Object],
       range: [Array],
       raw: '<strong>HTML と、Page クラスに Footer 部とする Container を追加する</strong>' },
     type: 'UNKNOWN' ],
  loc: 
   { start: { line: 1144, column: 36 },
     end: { line: 1144, column: 104 } },
  range: [ 46454, 46522 ],
  raw: '<p><strong>HTML と、Page クラスに Footer 部とする Container を追加する</strong></p>' }
```

In the case of Markdown, the data of the node is as follows.

```console
{ type: 'Paragraph',
  children: 
   [ { type: 'Code',
       value: 'o',
       loc: [Object],
       range: [Array],
       raw: '`o`' },
     { type: 'Str',
       value: ' をつけたルールを一律 グローバルインストールします。    ',
       loc: [Object],
       range: [Array],
       raw: ' をつけたルールを一律 グローバルインストールします。    ' } ],
  loc: 
   { start: { line: 117, column: 0 },
     end: { line: 117, column: 34 } },
  range: [ 5761, 5795 ],
  raw: '`o` をつけたルールを一律 グローバルインストールします。    ' }
```

I thought false detection occurred because HTML tags included in the `raw` property of the Paragraph node generated from HTML.
I attempted to fix it. but I could not decide whether to remove HTML tags on the rule side or whether HTML parser should remove the HTML tag of `raw` property.

--- a/package.json
+++ b/package.json
@@ -44,6 +44,7 @@
   "dependencies": {
     "object-assign": "^4.0.1",
     "sentence-splitter": "^2.0.0",
-    "textlint-rule-helper": "^2.0.0"
+    "textlint-rule-helper": "^2.0.0",
+    "textlint-util-to-string": "^2.1.1"
   }
 }

--- a/src/no-start-duplicated-conjunction.js
+++ b/src/no-start-duplicated-conjunction.js
@@ -61,7 +61,8 @@ module.exports = function(context, options = {}) {
                     // adjust index 
                     // if  "また、[import, a.js](a.js)" then originalIndex is used.
                     // if "[import, binary-example.js](src/binary-example.js)" then  originalIndex is undefined.
-                    if (!ignoreNodeManager.isIgnoredIndex(typeof originalIndex !== "undefined" ? originalIndex : sentenceStartIndex)) {
+                    const targetIndex = typeof originalIndex !== "undefined" ? originalIndex : sentenceStartIndex;
+                    if (!ignoreNodeManager.isIgnoredIndex(targetIndex)) {
                         report(node, new RuleError(`Don't repeat "${phrase}" in ${options.interval} phrases`, {
                             line: Math.max(sentence.loc.start.line - 1, 0),
                             column: sentence.loc.start.column

--- a/src/no-start-duplicated-conjunction.js
+++ b/src/no-start-duplicated-conjunction.js
@@ -4,6 +4,7 @@ import {RuleHelper, IgnoreNodeManager} from "textlint-rule-helper";
 import ObjectAssign from "object-assign";
 const splitSentence = require("sentence-splitter").split;
 const SentenceSyntax = require("sentence-splitter").Syntax;
+import StringSource from "textlint-util-to-string";
 const defaultOptions = {
     interval: 2
 };
@@ -39,7 +40,8 @@ module.exports = function(context, options = {}) {
                 return;
             }
             ignoreNodeManager.ignoreChildrenByTypes(node, ignoreTypes);
-            const text = getSource(node);
+            const source = new StringSource(node);
+            const text = source.toString();
             const sentences = splitSentence(text, {
                 charRegExp: /[。]/
             }).filter(sentence => {
@@ -55,7 +57,11 @@ module.exports = function(context, options = {}) {
                 }
                 if (useDuplicatedPhase) {
                     const sentenceStartIndex = node.range[0] + sentence.range[0];
-                    if (!ignoreNodeManager.isIgnoredIndex(sentenceStartIndex)) {
+                    const originalIndex = source.originalIndexFromIndex(sentenceStartIndex);
+                    // adjust index 
+                    // if  "また、[import, a.js](a.js)" then originalIndex is used.
+                    // if "[import, binary-example.js](src/binary-example.js)" then  originalIndex is undefined.
+                    if (!ignoreNodeManager.isIgnoredIndex(typeof originalIndex !== "undefined" ? originalIndex : sentenceStartIndex)) {
                         report(node, new RuleError(`Don't repeat "${phrase}" in ${options.interval} phrases`, {
                             line: Math.max(sentence.loc.start.line - 1, 0),
                             column: sentence.loc.start.column

